en/admin/config.rst

```diff
@@ -1959,6 +1963,23 @@ The following are other parameters. The type and value range for each parameter
         java_stored_procedure_port=4334
         .....
 
+**java_stored_procedure_jvm_options**
+
+    **java_stored_procedure_jvm_options** is a parameter to configure Java Virtual Machine (JVM) and Java options on which Java stored procedures are executed. Each option string should be separated by spaces. For JVM options, there are three types of options; standard, non-standard and advanced options. non-standard and advanced options are not guaranteed to be supported on all VM implementations. If the parameter value configured in [@<database>], it overwrites the value specified in [common]. ::
+
+        ..... 
+        [common] 
+        ..... 
+        java_stored_procedure_jvm_options="-Xms1024m -Xmx1024m -XX:PermSize=512m -XX:MaxPermSize=512m"
+        .....
+        [@testdb]
+        .....
+        java_stored_procedure=yes
+
+        # Note that -XX:PermSize=512m and -XX:MaxPermSize=512m will not be applied for testdb, Even though they specified in common section.
+        java_stored_procedure_jvm_options="-Xms2048m -Xmx2048m"
```

Comment:
"[common]" looks better than "common" in  the sentence _"# Note that -XX:PermSize=512m and -XX:MaxPermSize=512m will not be applied for testdb, Even though they specified in **common** section."_